pages/api/entity_types/[slug]/icon_change.js

```diff
@@ -0,0 +1,51 @@
+import EntityType from "@backend/models/core/EntityType";
```

Comment:
Could this be refactored in `pages/api/entity_types/[slug].js`, in together with the other attributes in the PUT section?